civil_society_vote/hub/management/commands/init.py

```diff
@@ -30,6 +30,9 @@
 
 Ne pare rau, dar cererea de inscriere a organizatiei "{{ name }}" in platforma VotONG a fost respinsa.
 
+Motivul respingerii:
+{{ rejection_message }}
```

Comment:
By doing this, we are burdening the Committee to remember what else is inside the e-mail body, aside from the text they write. 

Let's make this more simple. 
Let's allow the Committee member to write the entire e-mail in the editor. That way, they control the greeting, the ending, the way the information is presented. And they don't need to remember the template.

Reply:
Done. You only need to have `{{ rejection_message }}` in the production email template.